### DISCUSSION TITLE
Ability to modify layout and exiting props of a mounted component

### DISCUSSION THREAD

**Discussion by lcarrettin** [software-mansion/react-native-reanimated#5193] [Ideas]
I was looking for a way to modify the layout and exiting props of the elements of a list (as part of an app’s theme change) without having to write overly complicated and performance impacting custom animations,  but it seems that these props are not modifiable.
I had a look at the createAnimatedComponent source code and such animations are configured inside _setComponentRef which is triggered by the ref prop assignment that is called only on mount/unmount.

Shouldn’t Layout Animations props behave as all the other props?

**Comment by tomekzaw** [comment]
@lcarrettin thanks for the suggestions, @bartlomiejbloniarz is working on it!

**Reply by lcarrettin** [reply]
Thanks!
